lib/node/fullnode.js

```diff
@@ -147,24 +151,28 @@ class FullNode extends Node {
       cors: this.config.bool('cors')
     });
 
-    this.ns = new RootServer({
-      logger: this.logger,
-      key: this.identityKey,
-      host: this.config.str('ns-host'),
-      port: this.config.uint('ns-port', this.network.nsPort),
-      lookup: key => this.chain.db.tree.get(key),
-      publicHost: this.config.str('public-host')
-    });
-
-    this.rs = new RecursiveServer({
-      logger: this.logger,
-      key: this.identityKey,
-      host: this.config.str('rs-host'),
-      port: this.config.uint('rs-port', this.network.rsPort),
-      stubHost: this.ns.host,
-      stubPort: this.ns.port,
-      noUnbound: this.config.bool('rs-no-unbound')
-    });
+    if (!this.noDNS) {
```

Comment:
nit: because this.noDNS and noRS are not used outside you could inline this.config thing here)